apps/artwork/components/partner_artworks/query.coffee

```diff
@@ -6,7 +6,7 @@ module.exports = """
       counts {
         artworks(format: "0,0", label: "work")
       }
-      artworks(size: 20, sort:published_at_desc, exclude: [$id]){
+      artworks(size: 20, sort:published_at_desc, for_sale:true, exclude: [$id]){
```

Comment:
yes, this should maybe be PR'd separately as it is old and unrelated. the point was to limit the related artworks from the same partner on the artwork page to only ones for sale.